### PR TITLE
AboutMe page

### PR DESCRIPTION
**Pull request by sophia-qin**
Added datastore logic, aboutme servlet, and edited html to get basic aboutMe textbox loading.

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+     * Responds with the "about me" section for a particular user.
+     */
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws IOException {
+
+        response.setContentType("text/html");
+
+        String user = request.getParameter("user");
+
```
Comment (tylerhou) [inline src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: Here I think the line spaces make the code a bit harder to read. I recommend finding digestible, logical chunks of code and separating them with line breaks. Something like:

```java
         response.setContentType("text/html");

         String user = request.getParameter("user");
         if (user == null || user.equals("")) {
            // Request is invalid, return empty response
            return;
         }

         User userData = datastore.getUser(user);
         if (userData == null || userData.getAboutMe() == null) {
            return;
         }
```

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        if(user == null || user.equals("")) {
```
[the remark this answers is not in the document]
Comment (tylerhou) [reply src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: Also, prefer `user.isEmpty()`.

[the remark this answers is not in the document]
Comment (tylerhou) [reply src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: (I see that this code + formatting came from the codeustudents documentation; I'll send them fixes.)

File: src/main/webapp/user-page.html [hunk 3]
```diff
@@ -43,5 +54,13 @@ <h1 id="page-title">User Page</h1>
 
     <div id="message-container">Loading...</div>
 
+
```
Comment (tylerhou) [inline src/main/webapp/user-page.html]: Please remove this extra whitespace (and above on line 44/45)

**Comment by jajanet** [comment]
with the conflict merge, do you want to approve the other pull request instead? is this changing the same thing?

**Pull request closed without merging**